app/src/main/AndroidManifest.xml

```diff
@@ -153,4 +154,4 @@
       android:name=".walkthrough.WalkthroughActivity"
       android:theme="@style/OppiaThemeWithoutActionBar" />
   </application>
-</manifest>
+</manifest>
```

Comment:
EOF is missing, add one blank line at the end.

Reply:
Done